src/GoCardless.js

```diff
@@ -70,7 +77,101 @@ export default class GoCardless {
         };
         let path = '/redirect_flows/' + redirectFlowId + '/actions/complete';
         let options = buildOptions(this.token, this.endPoint, path, 'POST', body);
+        return goCardlessRedirectRequest(options);
+    }
+
+    /**
+     * Sends a request for payment creation
+     * https://developer.gocardless.com/pro/2015-07-06/#payments-create-a-payment
+     * @param mandateID REQUIRED ID of the mandate against which payment should be collected
+     * @param amount REQUIRED amount in pence, cents or öre
+     * @param charge_date a future date on which the payment should be collected
+     * @param currency defaults to EUR, either EUR, GBP or SEK
+     * @param description human readable description sent to payer
+     * @param metadata any data up to 3 pairs of key-values
+     * @param internalReference your own internal reference
+     */
+    createPayment(mandateID, amount, currency = "EUR", charge_date = null, description = null, metadata = null, internalReference = null) {
+
+        const body = {
+            'payments': {
+                'amount': amount,
+                'currency': currency,
+                'charge_date': moment(charge_date, 'YYYY-MM-DD'),
```

Comment:
can we remove that dependency on momentjs?
wanna keep that lib as slim as possible